api/pom.xml

```diff
@@ -259,7 +259,7 @@
                             <header><![CDATA[<br>Jakarta Server Pages API v${project.version}]]></header>
                             <bottom><![CDATA[
 Comments to: <a href="mailto:jsp-dev@eclipse.org">jsp-dev@eclipse.org</a>.<br>
-Copyright &#169; 2018, 2020 Eclipse Foundation. All rights reserved.<br>
+Copyright &#169; 2018, 2021 Eclipse Foundation. All rights reserved.<br>
```

Comment:
minor but date should be 2022

Reply:
2021 is correct. See the associated commit comment.